firefox_accounts/firefox_accounts.model.lkml

```diff
@@ -11,7 +11,7 @@ explore: +growth_accounting {
   join: fxa_first_seen_table {
     fields: []
     relationship: many_to_one
-    sql_on: ${growth_accounting.user_id} = ${fxa_first_seen_table.user_id} ;;
```

Comment:
An alternative option here would be to add a new dimension here:
https://github.com/mozilla/looker-spoke-default/blob/main/firefox_accounts/views/growth_accounting.view.lkml#L15

```
  dimension: user_id {
    sql: ${TABLE}.user_id_sha256 ;;
    type: STRING
  }
```

And hide the sha256 field:
```
dimension: user_id_sha256 {
    hidden: yes
}
```
Or even aliasing `user_id_sha256` AS `user_id` in the BQ view.